dataland-frontend/tests/e2e/specs/low/pages/VisitPagesUnauth.ts

```diff
@@ -2,7 +2,7 @@ describe("Test that if unauthenticated will be redirected to login page", () =>
   it("test for each of given paths", () => {
     const pages = ["/upload", "/search", "/searchtaxonomy", "/companies/:companyID/eutaxonomies"];
     pages.forEach((page) => {
-      cy.visitAndCheckAppMount(page);
```

Comment:
Warum die Änderung?

Reply:
Weil die Weiterleitung auf die Login-Seite bei mir zu schnell ging, und somit cypress die #app Komponente nicht finden kann (weil er sofort auf der login Seite ist). Deswegen habe ich das umgeschrieben. In der CI funktionierts allerdings auch mit checkAppMount, wiel da die latenz höher ist.

Reply:
Das ist ja nervig. Habe lokal aber auch das Problem, dass bei dem Cypress Test "SearchTaxonomy" der Tooltip Test failed, da die Suchvorschläge noch offen sind. Frage ist ob man die Testreihenfolge einfach ändert, damit die sich nicht in die Quere kommen.

Reply:
Könnte man überlegen. Aber ich glaube hier muss ja auch nicht auf die #app Komponente geprüft werden. Hier ist ja das einzig wichtige, dass man auf das Login weitergeleitet wird. Deswegen finde ich die Änderung hier so oder so okay. Oder was meinst du?

Reply:
Passt für mich